README.md

```diff
@@ -1,12 +1,12 @@
-# docs
+# Docs
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/f1919fde-0ef7-4b52-884f-870e354f74e4/deploy-status)](https://app.netlify.com/sites/discordtickets/deploys)
+[![Netlify Build Status](https://api.netlify.com/api/v1/badges/f1919fde-0ef7-4b52-884f-870e354f74e4/deploy-status)](https://app.netlify.com/sites/discordtickets/deploys)
 
-This is the source for the Discord Tickets main/documentation website.
-You can view the live site at [discordtickets.app](https://discordtickets.app).
+This is the source code for Discord Tickets fourth iteration documentation.
+You can view the latest build of the V4 site [on Netlify](https://v4--discordtickets.netlify.app/).
```

Comment:
Unnecessary and will soon be reverted so no point changing

Reply:
ok